### PR TITLE
Feature: Add option to auto-expand category

### PR DESCRIPTION
**Pull request by p1r9t3**

### CODE REVIEW

File: preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java [hunk 1]
```diff
@@ -253,4 +264,8 @@ public ReadOnlyStringProperty descriptionProperty() {
   public Node getItemIcon() {
     return itemIcon;
   }
+
+  public boolean autoExpand() {
```
Comment (martinfrancois) [inline preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java]: Can you please add javadoc here?

File: preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java [hunk 2]
```diff
@@ -253,4 +264,8 @@ public ReadOnlyStringProperty descriptionProperty() {
   public Node getItemIcon() {
     return itemIcon;
   }
```
Comment (martinfrancois) [inline preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java]: I think I would prefer `isExpand` for the getter, to make it obvious

File: preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java (hunk 1 above)
Comment (p1r9t3) [reply preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java]: Done.

File: preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java (hunk 2 above)
Comment (p1r9t3) [reply preferencesfx/src/main/java/com/dlsc/preferencesfx/model/Category.java]: Done.

**Comment by p1r9t3** [comment]
> Hi, thanks for the contribution! This feature is a nice addition in my opinion!
> Can you please add an example on how to use it to the demos and extend the Readme to contain an explanation of this feature?

Done and submitted.  I added to the README and to some of the demos.

**Pull request merged**